docs/en/running-node/monerod-tori2p.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Tor and I2P
+
+config: (../interacting/monero-config-file.md#monerodconf)
+---
+
+!!! note ""
+    How to add Tor and/or I2P to your Monero node
+
+### Assumptions
+
+You possess:
+
+- Basic understanding of Linux administration
+- Root access to a Linux distribution
+- A Monero Node
+
+Some commands assume Ubuntu but you can trivially translate them to your distribution.
+
+??? question "Why use anonymity networks?"
+
+    **Why use anonymity networks?**
+    You will be able to connect your desktop and mobile Monero wallets to your own trusted Monero node,
+    in a secure and private way over Tor or I2P.
+
+    **Tor and I2P hidden services for wallet interface** are useful for wallet users because it bypasses NATA and also works to mitigates MiTM risks (which are very real). Hidden service connections are end-to-end encrypted and private by default.
```

Comment:
```suggestion
    **Tor and I2P hidden services for wallet interface** are useful for wallet users because it bypasses [NAT](https://tailscale.com/blog/how-nat-traversal-works) and also works to mitigate MITM risks (which are very real). Hidden service connections are end-to-end encrypted and private by default.
```